src/test/java/com/cognifide/aemrules/htl/checks/CapitalizationCheckTest.java

```diff
@@ -0,0 +1,33 @@
+/*-
+ * #%L
+ * AEM Rules for SonarQube
+ * %%
+ * Copyright (C) 2015 Cognifide Limited
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.cognifide.aemrules.htl.checks;
+
+import com.cognifide.aemrules.htl.AbstractBaseTest;
+import org.junit.Test;
+
+public class CapitalizationCheckTest extends AbstractBaseTest {
```

Comment:
Please keep naming convention.
```suggestion
public class CamelCaseCheckTest extends AbstractBaseTest {
```

Reply:
fixed